sequence_task_3/student/src/impl.cpp

```diff
@@ -1,2 +1,23 @@
 #include "impl.h"
 
+#include <vector>
+
+std::string reverseWords(const std::string& input)
+{
+    std::string ret;
+    if(!input.empty())
```

Comment:
Will be much simpler:
```suggestion
    if (input.empty()) {
        return input;
    }
```